names.txt

```diff
@@ -1,3 +1,4 @@
 Hadi Hajihosseini 😎🙏🏻
 Hadi
 Fardin
+Mohammad Ab.
```

Comment:
Hello, please just add your first name to the end of the file!

Reply:
Hello, Sure